backends/cadence/aot/compiler.py

```diff
@@ -51,12 +58,32 @@ def export_to_edge(
 
     # Call to_edge to convert the graph to edge IR.
     edge_prog_manager = to_edge(
-        expo_program, compile_config=EdgeCompileConfig(_check_ir_validity=False)
+        expo_program,
+        compile_config=EdgeCompileConfig(
+            _check_ir_validity=False, _skip_dim_order=True
+        ),
     )
 
     if dump_graphs:
         logging.info(
             f"Edge graph:\n{edge_prog_manager.exported_program().graph_module.graph}"
         )
 
-    return edge_prog_manager, expo_program
+    return edge_prog_manager
+
+
+# Export the model and lower it to an EdgeProgramManager (in edge IR), and
+# apply passes specific to Cadence DSP execution.
+def export_to_cadence(
+    model: torch.nn.Module,
+    inputs: Tuple[Any, ...],
+    dump_graphs: bool = False,
+) -> EdgeProgramManager:
+    edge_program_manager = export_to_edge(model, inputs)
```

Comment:
Should this just take in an EdgeProgramManager that separately had `export_to_edge` called so that the APIs can be combined?